operations/update.go

```diff
@@ -212,24 +199,24 @@ func checkUpdate(client client.Communicator, silent bool) (updateStatus, error)
 	// server says is the latest
 	clients, err := client.GetClientConfig(context.Background())
 	if err != nil {
-		fmt.Fprintf(outLog, "Failed checking for updates: %v\n", err)
+		fmt.Fprintf(outLog, "Failed checking for updates: %s\n", err)
```

Comment:
can't we do some grip.NoticeWhen(!silent, message.WrapError()) thing

Reply:
We use this branch to return though.

Reply:
kinda? I did a thing